src/aerovaldb/jsondb/jsonfiledb.py

```diff
@@ -7,6 +7,7 @@
 from hashlib import md5
 from pathlib import Path
 from typing import Any, Awaitable, Callable
+from warnings import warn
```

Comment:
`warnings` is rather for low-level code. aerovaldb is used within code written with `logging`. Please use logger.warning instead.